utility_analysis_new/dpengine.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""DPEngine for utility analysis."""
+
+import pipeline_dp
+from pipeline_dp import contribution_bounders
+import utility_analysis_new.contribution_bounders as utility_contribution_bounders
+from dataclasses import dataclass
+
+
+class DPEngine(pipeline_dp.DPEngine):
```

Comment:
I'm thinking, it might be confusing that both classes called `DPEngine`, maybe `UtilityAnalysisEngine`, WDYT?

Reply:
This is the same case as ContributionBounders, no? I think we can use the package/module path to tell the difference between regular dp_engine and utility_analyis.dp_engine.

Reply:
ContributionBounders are less confusing:
1. They are not part of public API, so no user confusion
2.`class DPEngine(pipeline_dp.DPEngine):` the line is especially confusing (`DPEngine` is subclass of `DPEngine`)
So I think `UtilityAnalysisEngine` or something like that would be much better.